ndb/tests/unit/test_model.py

```diff
@@ -2653,12 +2655,14 @@ class TimeTravelVehicle(model.Model):
             TimeTravelVehicle(speed=28)
 
     @staticmethod
+    @pytest.mark.usefixtures("reset_kind_map")
```

Comment:
Sorry I'm just checking these, but this test doesn't actually create a `Model` subclass, so why is the kind map reset?

Reply:
Copy pasta from splitting the tests up.